tutorials/install-cockroachdb-scaleway-instances/index.mdx

```diff
@@ -0,0 +1,480 @@
+    ```
+    scw instance server create name=db01 zone=fr-par-1 type=PRO2-XXS image=ubuntu_jammy ip=none additional-volumes.0=b:30G
+    ```
+
+6. Create a private NIC (Network Interface Card) for your Instance.
+    ```
+    scw instance private-nic create zone=fr-par-1 server-id=<id> private-network-id=<id>
+    ```
+
+7. Repeat steps 5 and 6 to create the two other nodes in the `fr-par-2` and `fr-par-3` AZs.
+
+4. Check your Instance's DHCP entry on the Private Network and retrieve its private IP.
+    ```
+    scw vpc-gw dhcp-entry list gateway-network-id=<id>
+    ```
+
+5. Connect to the Instances using the gateway bastion.
+    ```
+    ssh -J bastion@<gw-ip>:61000 root@db01.priv
+    ```
+
+## Synchronizing clocks
+
+CockroachDB requires moderate levels of [clock synchronization](https://www.cockroachlabs.com/docs/v23.1/recommended-production-settings#clock-synchronization) to preserve data consistency.
+
+For this reason, nodes spontaneously shut down when they detect that their clock is out of sync with at least half of the other nodes in the cluster by 80% of the maximum offset allowed (500ms by default).
+
+This is done to avoid consistency anomalies. However, it is best to prevent clocks from drifting too far off. To do so, a clock synchronization software can be run on each mode.
+
+For the sake of this tutorial, we will be using [NTP](https://www.ntp.org/documentation/4.2.8-series/) to keep offsets in single-digit milliseconds.
+
+<Message type="note">
+  Note that other methods of clock synchronization are suitable as well.
+</Message>
+
+1. Connect to one of your nodes using SSH.
+
+2. Disable `timesyncd` as it tends to be active by default on some Linux distributions by running the following command.
+    ```
+    timedatectl set-ntp no
+    ```
+
+    <Message type="note">
+        You can also make sure `timesyncd` is off by running the `timedatectl` command.
+    </Message>
+
+3. Run the following command to install the NTP package.
+    ```
+    apt-get install ntp -y
+    ```
+
+4. Run the following command to stop the NTP daemon.
+    ```
+    service ntp stop
+    ```
+
+5. Configure the machine's clock with Google's NTP service, in the `/etc/ntp.conf` file, remove or comment out any lines starting with server or pool and add the following lines.
+    ```
+    server time1.google.com iburst
+    server time2.google.com iburst
+    server time3.google.com iburst
+    server time4.google.com iburst
+    ```
+
+6. Restart the NTP daemon:
+    ```
+    service ntp start
+    ```
+    <Message type="important">
+      We recommend Google's NTP service because it handles "smearing" the leap second. If you use a different NTP service that does not smear the leap second, be sure to configure client-side smearing in the same way on each machine. See the [production checklist](https://www.cockroachlabs.com/docs/v23.1/recommended-production-settings) for details.
```

Comment:
```suggestion
      We recommend using Google's NTP service because it handles "smearing" the leap second. If you use a different NTP service that does not smear the leap second, be sure to configure client-side smearing in the same way on each machine. See the [production checklist](https://www.cockroachlabs.com/docs/v23.1/recommended-production-settings) for details.
```